src/components/DateRangePicker.jsx

```diff
@@ -138,7 +138,9 @@ export default class DateRangePicker extends React.Component {
 
   onEndDateChange(endDateString) {
     const { startDate, isOutsideRange, onDatesChange, onFocusChange, displayFormat } = this.props;
-    const endDate = toMomentObject(endDateString, displayFormat);
+
+    const formatString = typeof displayFormat === 'function' ? displayFormat() : displayFormat;
```

Comment:
Okay. I wasn't sure if it made more sense to have it fail abruptly in this way or fail silently, but still function, in the other. I can change it to the former.